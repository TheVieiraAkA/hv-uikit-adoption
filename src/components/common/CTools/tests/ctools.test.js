/**
 * Copyright (c) 2020 Hitachi Vantara Corporation.
 *
 *  The copyright to the computer software herein is the property of
 *  Hitachi Vantara Corporation. The software may be used and/or copied only
 *  with the written permission of Hitachi Vantara Corporation or in accordance
 *  with the terms and conditions stipulated in the agreement/contract
 *  under which the software has been supplied.
 */

/* eslint-env jest */

import { mount } from "enzyme";
import React from "react";
import toJson from "enzyme-to-json";
import Ctools from "../index";

describe("CTools", () => {
  let wrapper;

  describe("index", () => {
    beforeEach(() => {
      wrapper = mount(<Ctools cdfEmbedPath="" dashboardPath="" />);
    });

    it("should match snapshot", () => {
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
