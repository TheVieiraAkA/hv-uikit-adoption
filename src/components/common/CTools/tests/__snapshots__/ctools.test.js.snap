// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CTools index should match snapshot 1`] = `
<LoadableComponent
  cdfEmbedPath=""
  dashboardPath=""
>
  <loading
    error={null}
    isLoading={true}
    pastDelay={false}
    retry={[Function]}
    timedOut={false}
  >
    <DefaultLoading>
      <p>
        loading...
      </p>
    </DefaultLoading>
  </loading>
</LoadableComponent>
`;
